test/e2e/kubevirt.go

```diff
@@ -81,6 +81,23 @@ passwd:
   - name: core
     password_hash: $y$j9T$b7RFf2LW7MUOiF4RyLHKA0$T.Ap/uzmg8zrTcUNXyXvBvT26UgkC6zZUVg3UKXeEp5
 `
+		labelNode = func(nodeName, label string) error {
```

Comment:
can this cosmetics be split please to another commit ? (also the usage of it of course)

Reply:
is not a super big refactoring, better let's keep it at the same commit.